bin/i18n/utils/sync_out_base.rb

```diff
@@ -43,7 +43,7 @@ def perform
         I18nScriptUtils.process_in_threads(languages) do |language|
           process(language)
         ensure
-          I18nScriptUtils.remove_empty_dir I18nScriptUtils.locale_dir(language[:crowdin_name_s])
+          I18nScriptUtils.remove_empty_dir I18nScriptUtils.crowdin_locale_dir(language[:locale_s])
```

Comment:
Synced down files go to a temporal directory `i18n/crowdin` instead of `i18n/locales`